app/models/agent.rb

```diff
@@ -100,6 +100,10 @@ def receive_web_request(params, method, format)
     ["not implemented", 404]
   end
 
+  # alternate method signature for receive_web_request
```

Comment:
I think it helps anyone reading the code to see that there's an alternate way for them to define `receive_web_request` but if you think it muddies things up too much I'll remove it.